### DISCUSSION TITLE
Setup android environment for contributions

### DISCUSSION THREAD

**Discussion by SkySails** [rnmapbox/maps#2303] [Q&A] [answered]
As part of #2294 , I'd like to setup the project stored in [`/android/rctmgl`](https://github.com/rnmapbox/maps/tree/main/android/rctmgl) in android studio to make contribute to the v10 project easier.

However, when running a Gradle sync I'm met with this message:
```
A problem occurred evaluating root project 'rctmgl'.
> Plugin with id 'com.android.library' not found.
```

I tried looking for contribution guides, but none of the documents that I could find in this repository seem to be aimed towards helping to set up the native projects. I would love to get some help with this!

Perhaps more thorough contribution guides would lower the threshold a bit and increase the number of contributions as well! 😄

**Comment by mfazekas** [comment, marked as answer]
You should be able to open example/android from android studio.

**Reply by SkySails** [reply]
Yes I am! I though that it would only let me modify the android part of the example RN project, but now that I look again I can clearly see an `rctmgl` folder there as well. I must've missed it the first time, probably because I didn't find that folder from VSCode.

Thanks for the pointer! All looks good now!
